i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md

```diff
@@ -17,6 +17,38 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
 
 ## Centreon MAP
 
+### 23.04.1
+
+Release date: `July 20, 2023`
+
+#### Enhancements
+
+- [Licensing] Improved automatic license renewal.
+- [UI] Make metric link modal labels more explicit by replacing "metric 1" and "metric 2" by "metric in" and "metric out".
+- [UI] You can now hide the label of a resource.
+- [UX] You can now duplicate maps.
+- [Viewer] You can now define a label for a URL shape.
+- [Widget] Fixed incorrect behavior when saving the MAP widget.
+
+#### Bug fixes
+
+- [Editor] Fixed a bug where empty metric names were blocking edition.
```

Comment:
replace with [Editor] Fixed a bug where empty metric min or max prevent to save a map.
(sorry for the late change)